src/OrchardCore.Modules/OrchardCore.Lucene/Views/Items/LuceneIndexRebuildDeploymentStep.Fields.Edit.cshtml

```diff
@@ -0,0 +1,51 @@
+@model LuceneIndexRebuildDeploymentStepViewModel
+
+<h5>@T["Rebuild Search Indexes"]</h5>
```

Comment:
Also do this in the other two places.

```suggestion
<h5>@T["Rebuild Lucene Search Indexes"]</h5>
```

Reply:
Consistency. We should use indices or indexes everywhere. I personally use indices most of the time.